addons/binding/org.openhab.binding.tado/src/main/api/pom.xml

```diff
@@ -55,11 +56,29 @@
 				</dependencies>
 			</plugin>
 
+			<plugin>
```

Comment:
That's a bit unexpected. Why is this part changed?

Reply:
This pom is used to generate the jar in the lib folder in this project. This change allows for generating of a jar version that is OSGi compatible so no need to wrap the library. This change includes a newly generated jar file from this changed pom.